src/buttonGroup/index.tsx

```diff
@@ -34,7 +36,23 @@ const ButtonGroup = ({
   ]
   return (
     <div className={cc(classNames)}>
-      {children}
+      {React.Children.toArray(children).map((button, index) => {
```

Comment:
can you .filter(button => React.isValidElement(button)) ?
instead of the if statement below ?